src/views/Pathways/Edit/Course/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect } from 'react';
+import { bindActionCreators } from 'redux';
+import { withRouter } from 'react-router';
+import { compose } from 'recompose';
+import { connect } from 'react-redux';
+import { useSnackbar } from 'notistack';
+import { withTheme } from '@material-ui/core';
+import {find} from 'lodash';
+import { selectors, setCourseToView, addOrEditCourse } from '../../store';
+import { selectors as layoutSelectors, setRightPaneLoading } from '../../../../layouts/TwoColumn/store';
+import { ngFetch } from '../../../../providers/NGFetch';
+import { getCourseEditFormStructure } from '../../forms/course';
+import FormBuilder from '../../../../components/FormBuilder';
+import Spacer from '../../../../components/Spacer';
+import RightPaneWithTitle from '../../../../components/RightPaneWithTitle';
+import history from '../../../../providers/routing/app-history';
+
+const PathwayCourseEdit = ({ rightPaneLoading, actions, match, theme }) => {
+
+  const { PathwayCourseId, pathwayId } = match.params;
+  const { enqueueSnackbar } = useSnackbar();
+ 
+
+  const [CoursesAlreadyPresent, setCoursesAlreadyPresent] = React.useState(null);
+  useEffect(() => {
+    const fetchallCourses = async () => {
```

Comment:
Similar to the add screen. Also because it is exactly the same logic in the add form too, how can we modularize it too?